flink-connector-postgres-cdc/src/main/java/com/alibaba/ververica/cdc/connectors/postgres/table/PostgreSQLTableFactory.java

```diff
@@ -83,6 +83,12 @@
 			"Supported values are decoderbufs, wal2json, wal2json_rds, wal2json_streaming,\n" +
 			"wal2json_rds_streaming and pgoutput.");
 
+	public static final ConfigOption<Boolean> IGNORE_ON_NOCHANGE = ConfigOptions
+			.key("ignore-on-nochange")
+			.booleanType()
+			.defaultValue(false)
+			.withDescription("optional flag whether to ignore the record if the fields have no change or not.");
```

Comment:
I think maybe we don't need an option for this. This is an optimization which can be enabled all the time.

Reply:
I agree with this. I will re-optimize the code, and add a integration test for this.